DTAdmin/Student/Controllers/StudentViewController.swift

```diff
@@ -120,11 +120,14 @@ extension StudentViewController: UITableViewDataSource {
         cell.name.text = filteredList[indexPath.row].studentName
         cell.fName.text = filteredList[indexPath.row].studentFname
         cell.sName.text = filteredList[indexPath.row].studentSurname
+//        if let photo = filteredList[indexPath.row].photo {
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.